feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/adapter/MealsAdapter.kt

```diff
@@ -67,7 +68,7 @@ class MealsAdapter(
             availableView.text = "Available: ${meal.availableFromDate}"
             expiryView.text = "Expires: ${meal.expiryDate}"
 
-            portionsView.text = getQuantityText(meal.quantity)
+            portionsView.text = "#    ${meal.quantity.getPortionsString()}"
```

Comment:
Lifted the quantity portions logic up to an Int extension in the shared layer.  If desired we could just return quantity as the corrected string.

Reply:
I think we need quantity for too many other things to just return the formatted string in the meal (like for whether the button is disabled, text colour etc).

With this being in here, we have Strings in a few separate places. In iOS we have a dedicated Strings module and so it's easy to find them if they need changing. In Android, it seems they are being set in the views (or viewModels) but then if this one bit of text is returned from an extension in the shared code it could be kind of confusing.

Maybe we should move all the Strings to shared code - and if we are going to do that then I think this is fine but otherwise I'm unsure on it

Reply:
We had issues accessing the Strings file from other modules when it's in the shared. So for now we have a Strings file within each module til we find a fix - is that still the case @dsmith42 ?

Reply:
I agree with your thinking.  For now I can move this into an extensions  on Android while we think about the other option.